package reinferio.saltfish;

import "source.proto";


message Response {
    enum Status {
	OK = 0;
	INVALID_SCHEMA = 1;
    }
    required Status status = 1;
    optional string msg = 2;
}


message Request {
    required string source_id = 1;
    repeated reinferio.source.Row rows = 2;
}


service SourceManager  {
    rpc push_rows(Request) returns (Response);
}